core/Cargo.toml

```
@@ -380,6 +380,7 @@ probe = { version = "0.5.1", optional = true }
 getrandom = { version = "0.2", features = ["js"] }
 
 [dev-dependencies]
+codspeed-criterion-compat = { version = "2.6.0", features = ["async", "async_tokio"] }
 criterion = { version = "0.5", features = ["async", "async_tokio"] }
```

You can rename the `criterion` package to `codpseed-criterion-compat` to avoid having to change all the `use` in the bench files:

```suggestion
criterion = { package="codspeed-criterion-compat", version = "2.7.2", features = ["async", "async_tokio"] }
```